firefox-ios/Client/Frontend/Browser/DownloadHelper/DownloadLiveActivityWrapper.swift

```diff
@@ -40,6 +40,14 @@ class DownloadLiveActivityWrapper: DownloadProgressDelegate {
         }
     }
 
+    func update() {
```

Comment:
Lets make this method private since it will only be called from within the wrapper class.